galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,161 @@
+                        'shape': {
+                            'title': 'Manage repositories in Private Automation Hub',
+                            'description': 'Add community and privately developed collections \
+                                to your Private Automation Hub.',
+                            'link': {
+                                'title': 'Get started',
+                                'external': True,
+                                'href': 'https://access.redhat.com/documentation/en-us/\
+                                    red_hat_ansible_automation_platform/1.2/html/\
+                                    publishing_proprietary_content_collections_in_\
+                                    automation_hub/index',
+                            },
+                        },
+                    },
+                ],
+                'learn': [
+                    {
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                        'shape': {
+                            'title': 'Connect Automation Hub to your automation infrastructure',
+                            'link': {
+                                'title': 'Get started',
+                                'external': True,
+                                'href': 'https://docs.ansible.com/ansible-tower/latest/html/userguide/\
+                                    projects.html?extIdCarryOver=true&\
+                                    sc_cid=701f2000001Css5AAC#using-collections-in-tower',
+                            },
+                        },
+                    },
+                    {
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                        'shape': {
+                            'title': 'Learn about namespaces',
+                            'description': 'Organize collections content into \
+                                namespaces users can access.',
+                            'link': {
+                                'title': 'Learn more',
+                                'external': True,
+                                'href': 'https://access.redhat.com/documentation\
+                                    /en-us/red_hat_ansible_automation_platform/1.2/html\
+                                    /curating_collections_using_namespaces_in_automation_hub/index',
+                            },
+                        },
+                    },
+                    {
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                        'shape': {
+                            'title': 'Explore Red Hat certified collections',
```

Comment:
I looked into it, but then it would be the same url then the other one. Or did you meant a different page than the `collections` page with the list of collections and the toolbar for search?